go-controller/hybrid-overlay/pkg/controller/master.go

```diff
@@ -179,30 +202,21 @@ func (m *MasterController) Add(node *kapi.Node) {
 		if allocatedSubnet != nil {
 			_ = m.releaseNodeSubnet(node.Name, allocatedSubnet)
 		}
-		klog.Errorf("failed to set hybrid overlay annotations for %s: %v", node.Name, err)
+		return fmt.Errorf("failed to set hybrid overlay annotations for %s: %v", node.Name, err)
 	}
+	return err
```

Comment:
@dave-tucker should be `return nil` I think? Shouldn't have any errors at this point; they've already all been returned. So you could change everything here to `if err := ` too if you want and get rid of the `var err error`

Reply:
done